VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.html

```diff
@@ -3,11 +3,11 @@
     <div class="govuk-!-margin-top-4">
       <div class="govuk-grid-column-one-third">
         <img
+          [src]="(hearingVenueIsInScotland$ | async) ? '/assets/images/scottish-crest-footer.png': '/assets/images/govuk-crest.png'"
```

Comment:
Should it use the image for the footer here? We should rename that if it's to be used other places.

Reply:
yea I will rename that to better reflect that image being used